.azure/azure-pipelines.ci.yml

```diff
@@ -330,6 +330,15 @@ stages:
   displayName: Build Linux - Non Tested
   dependsOn: []
   jobs:
+  - template: ./templates/build-config-user.yml
```

Comment:
I _think_ build is going to be enough. Opinions?

Reply:
build is enough for what? Releasing?

Reply:
No. To validate this new flag that only gets enabled for release builds otherwise.

Reply:
As far as I can tell, yes.